### DISCUSSION TITLE
Defaults and Automatic adjustment of settings.

### DISCUSSION THREAD

**Discussion by Jon-b-m** [Artificial-Pancreas/iAPS#282] [Ideas]
I have two propositions. 

1. Change the default iAPS settings to make them work for 80% of users. 

2. Monitor the statistics and insulin and make auto-adjustments to the iAPS settings (openAPS and oref2). Objectives are to improve already working settings (for the 80%) and to make them work also for the rest (the 20 %).

Above changes would allow us to have just two toggles in OpenAPS settings. 
* Use Automatic System settings
* View advanced settings 

Changing the defaults isn't hard, however auto tuning the settings will require some sound logic.

**Comment by xuan39846** [comment]
Good afternoon John, could you please provide the ipa file for the 2.3.0 build?
